setup.py

```diff
@@ -75,6 +75,7 @@
     zip_safe=False,
     platforms='any',
     install_requires=REQUIRES,
+    python_requires=">=3.4",
```

Comment:
'>=3.5.2'

Reply:
Excellent! Does #9328 need to be updated+closed then?

Reply:
You can update it

Reply:
I think aiohttp wants 3.5.3+ as minimum for the future.

Reply:
We are still Python 3.4 Pascal. We're waiting till aiohttp releases v3 to bump min requirement to 3.5

Reply:
Given this conversation, I don't think there is anything I need to do to get this PR ready.

Clearly it will be good to have the minimum python version explicitly stated in the codebase (setup.py).

Reply:
We already define the minimum required version in `homeassistant.const`, can you import it from there? We also have a different minimum version between Windows and other OSes (because of the asyncio loop that we use).

https://github.com/home-assistant/home-assistant/blob/7e246e4680d71fe9913678ede731e8c6b51aa7ae/homeassistant/const.py#L8-L9

Since we will soon align both min-versions, just sticking with 3.4.2 will do.